edx_solutions_api_integration/courses/views.py

```diff
@@ -1835,6 +1841,12 @@ def get(self, request, course_id):  # pylint: disable=W0613,W0221
 
 class CoursesMetricsCompletionsLeadersList(SecureAPIView):
     """
+    **DEPRECATION WARNING**
```

Comment:
Initially I thought we were supposed to deprecate the `UserGradeView` in `grades/api/views.py`, but then this part
> There is an API for that which already exists and is used by Apros (obviously), but there are two problems with the [existing API endpoints](https://github.com/edx-solutions/api-integration/blob/9d0f1a603e935f00bfd1b935e1738c663e4bf165/edx_solutions_api_integration/courses/views.py#L1722-L1790):
made me think these are the views that we need to deprecate because of their issues. If this is not the case, do you know which ones we should deprecate? Thanks!